src/pages/Home/components/Content/FilterBar.tsx

```diff
@@ -0,0 +1,36 @@
+import { Genre } from '../../../../shared/interfaces';
```

Comment:
So you could use aliases here right ?

Reply:
yes, but i will postpone it for now because the configuration of webpack is more difficult than I thought. it can be done by craco but it has a weird configuration api or make eject what i don't want to do now